packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -72,6 +60,31 @@ function mineOnEvent(contract: Contract) {
 
 beforeAll(async () => {
   provider = new providers.JsonRpcProvider(rpcEndpoint);
+
+  a = Wallet.create(
+    await ChainService.create({
```

Comment:
1. The original plan was for the `ChainService` to be an external component to the `server-wallet`, and writing it inside the `server-wallet` was only ever a short-term convenience.
2. Arguably it's easier to write the tests because it's easier to understand when you're using a mock and when you're expecting to use and test this external component which has special characteristics (i.e., connects to the internet).
3. I've already rewritten all of the tests and propagated all of the changes upstream for convenience, so I've made a best attempt to make the transition as smooth as possible.